datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java

```diff
@@ -104,6 +109,18 @@ public FlowIdentifier triggerSdxStopFlow(SdxCluster cluster) {
         return notify(selector, new SdxStartStopEvent(selector, cluster.getId(), userId));
     }
 
+    public String triggerDatalakeDatabaseBackupFlow(Long sdxId, String backupId, String backupLocation) {
```

Comment:
Why should FlowIdentifier return when it is not used? API invoking triggerDatalakeDatabaseBackupFlow uses operationId instead of flowIdentifier.

I can make changes to avoid the need for notifyDatabaseDrEvent. Does that address your concern?

Reply:
I have removed method "notifyDatabaseDrEvent" and re-used "notify".

Reply:
will update.